package.json

```diff
@@ -2,6 +2,6 @@
   "devDependencies": {
     "autoprefixer": "^10.4.0",
     "postcss": "^8.4.5",
-    "postcss-cli": "^8.3.1"
+    "postcss-cli": "^9.1.0"
```

Comment:
Looks like `dependabot` does not respect `^`:

>    ^: It will only do updates that do not change the leftmost non-zero number i.e there can be changes in minor version or patch version but not in major version. If you write ^13.1.0, when running npm update, it can update to 13.2.0, 13.3.0 even 13.3.1, 13.3.2 and so on, but not to 14.0.0 or above.

(from <https://nodejs.dev/learn/semantic-versioning-using-npm>)